Bottom: 3784a9081337bd31ddc7b6b2315291064c44cf51
Top:    30c50efcbff260548fc311d81a94810ad9b678ad
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 10:22:50 +0800

Refresh of debug.patch

---

diff --git a/modules/mount_autofs.c b/modules/mount_autofs.c
index c7f8b3d..3d96e69 100644
--- a/modules/mount_autofs.c
+++ b/modules/mount_autofs.c
@@ -184,7 +184,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 		return 1;
 	}
 	argv[0] = info->map;
-	error(LOGOPT_ANY, "info->type %s", info->type);
+	error(LOGOPT_ANY, "info type %s format %s map %s", info->type, info->format, info->map);
 
 	if (options) {
 		p = options;
